src/DssSpell.sol

```diff
@@ -61,140 +49,48 @@ contract DssSpellAction is DssAction {
     //
     // uint256 internal constant X_PCT_1000000003022265980097387650RATE = ;
 
+    // ---------- Math ----------
+    uint256 internal constant THOUSAND = 10 ** 3;
+    uint256 internal constant RAD      = 10 ** 45;
```

Comment:
Remove it after using DssExecLib for `vow.bump` change

Reply:
Amended in [0637ac4](https://github.com/makerdao/spells-mainnet/pull/414/commits/0637ac44647b0f65562432e977fdd71180ab273a)